hc/front/tests/test_add_channel.py

```diff
@@ -13,13 +15,14 @@ def test_it_adds_email(self):
 
         self.client.login(username="alice@example.org", password="password")
         r = self.client.post(url, form)
+        user = User.objects.get(email='alice@example.org')
 
         self.assertRedirects(r, "/integrations/")
         assert Channel.objects.count() == 1
```

Comment:
For asserting we use, more descriptive asserts provided by the testing client. For better documentation, you could change all asserts to their respective descriptive asserts from the testing client.

Reply:
Noted.